Cod3rsGrowth.Teste/TestesUnitarios/TesteRepositorios.cs

```diff
@@ -0,0 +1,123 @@
+    public TesteRepositorios()
+    {
+        filmeRepositorio = serviceProvider.GetService<IFilmeRepositorio>() ?? throw new Exception("Repositorio nao encontrado");
+        atorRepositorio = serviceProvider.GetService<IAtorRepositorio>() ?? throw new Exception("Repositorio nao encontrado");
+        usuarioRepositorio = serviceProvider.GetService<IUsuarioRepositorio>() ?? throw new Exception("Repositorio nao encontrado");
+    }
+
+    [Fact]
+    public void obter_todos_retorna_lista_de_todos_os_filmes_do_repositorio_quando_lista_nao_nula()
+    {
+        filmeRepositorio.Inserir(new Filme { Id = 1, Titulo = "De Volta Para o Futuro", Genero = GeneroEnum.Ficcao, Classificacao = ClassificacaoIndicativa.livre});
+        filmeRepositorio.Inserir(new Filme { Id = 2, Titulo = "Titanic", Genero = GeneroEnum.Romance, Classificacao = ClassificacaoIndicativa.doze});
+        filmeRepositorio.Inserir(new Filme { Id = 3, Titulo = "Star Wars", Genero = GeneroEnum.Ficcao, Classificacao = ClassificacaoIndicativa.livre});
+        filmeRepositorio.Inserir(new Filme { Id = 4, Titulo = "O Senhor dos Anéis: A Sociedade do Anel", Genero = GeneroEnum.Fantasia, Classificacao = ClassificacaoIndicativa.quatorze});
+        filmeRepositorio.Inserir(new Filme { Id = 5, Titulo = "O Senhor dos Anéis: As Duas Torres", Genero = GeneroEnum.Fantasia, Classificacao = ClassificacaoIndicativa.quatorze});
+        filmeRepositorio.Inserir(new Filme { Id = 6, Titulo = "O Senhor dos Anéis: O Retorno do Rei", Genero = GeneroEnum.Fantasia, Classificacao = ClassificacaoIndicativa.quatorze});
+        filmeRepositorio.Inserir(new Filme { Id = 7, Titulo = "Matrix", Genero = GeneroEnum.Ficcao, Classificacao = ClassificacaoIndicativa.dezesseis});
+        filmeRepositorio.Inserir(new Filme { Id = 8, Titulo = "Gladiador", Genero = GeneroEnum.Acao, Classificacao = ClassificacaoIndicativa.dezesseis});
+        filmeRepositorio.Inserir(new Filme { Id = 9, Titulo = "O Poderoso Chefão", Genero = GeneroEnum.Drama, Classificacao = ClassificacaoIndicativa.dezoito});
+        filmeRepositorio.Inserir(new Filme { Id = 10, Titulo = "Forrest Gump", Genero = GeneroEnum.Drama, Classificacao = ClassificacaoIndicativa.quatorze});
+        filmeRepositorio.Inserir(new Filme { Id = 11, Titulo = "Pulp Fiction", Genero = GeneroEnum.Acao, Classificacao = ClassificacaoIndicativa.dezoito});
+        filmeRepositorio.Inserir(new Filme { Id = 12, Titulo = "O Cavaleiro das Trevas", Genero = GeneroEnum.Acao, Classificacao = ClassificacaoIndicativa.quatorze});
+        var listaEsperada = TabelasSingleton.ObterInstanciaFilmes;
+        var titulo = listaEsperada[0].Titulo;
+        var id = listaEsperada[0].Id;
+        var genero = listaEsperada[0].Genero;
+        var classificacao = listaEsperada[0].Classificacao;
+
+        var lista = filmeRepositorio.ObterTodos();
+
+        Assert.NotEmpty(lista);
+        Assert.Equal(lista[0].Titulo, titulo);
+        Assert.Equal(lista[0].Id, id);
```

Comment:
aqui vc tá verificando apenas o primeiro elemento da lista, não tem necessidade se for assim, pode fazer só a verificação que a lista não está vazia e a quantidade de itens